File: web/js/games/dblclick.js
// Double-Click activity -- collect every themed picture with a double click.

import { TargetActivity } from "./targets.js";

export class DblClickActivity extends TargetActivity {
  static id = "dblclick";
  static suffix = "dblclick";
  static bg = "dblclick";
  static title = "Double click";
  static requireDouble = true;
}

File: src/games/dblclick.py
"""Double-Click activity - collect every themed picture with a double click."""

from __future__ import annotations

from .targets import TargetActivity


class DblClickActivity(TargetActivity):
    id = "dblclick"
    title = "Double click"
    background = "dblclick"
    themes_suffix = "dblclick"
    require_double_click = True
